src/classes/ADDR_Account_TDTM.cls

```diff
@@ -108,29 +108,44 @@ public class ADDR_Account_TDTM extends TDTM_Runnable {
 	                Boolean accAddrEnabledForRecType = UTIL_CustomSettingsFacade.getSettings().Accounts_Addresses_Enabled__c != null && acc.RecordTypeId != null
                                                        && UTIL_CustomSettingsFacade.getSettings().Accounts_Addresses_Enabled__c.contains(acc.RecordTypeId);
 	                
-	                if (isHousehold || accAddrEnabledForRecType) {
-	                    
-	                    // if the address info has been cleared
-	                    if(!ADDR_Addresses_UTIL.isAccAddressEmpty(accOld) && ADDR_Addresses_UTIL.isAccAddressEmpty(acc)) {
-	                        accsAddrInfoCleared.add(acc);
-	                        
-	                    // if the account address info has changed  
-	                    } else if (ADDR_Addresses_UTIL.isAccountAddressChanged(acc, accOld)) {
-	                        newAddrInfoAccs.add(acc);
-	                    }
-	                }
+                    if (!accountIdsUpdated.contains(acc.Id)){
```

Comment:
My concern here is: There is a scenario when an account with address is created, it will fire the insert trigger first and then create a new address record. After the new address is created, it will fire the update trigger to assign the address to the account. If we use one set of ids, update trigger will not be fired it seems like.